src/e2e/java/teammates/e2e/cases/e2e/InstructorCourseJoinConfirmationPageE2ETest.java

```diff
@@ -0,0 +1,63 @@
+package teammates.e2e.cases.e2e;
+
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.attributes.InstructorAttributes;
+import teammates.common.util.AppUrl;
+import teammates.common.util.Const;
+import teammates.common.util.StringHelper;
+import teammates.e2e.pageobjects.AppPage;
+import teammates.e2e.pageobjects.CourseJoinConfirmationPage;
+import teammates.e2e.pageobjects.ErrorReportingModal;
+import teammates.e2e.pageobjects.InstructorHomePage;
+
+/**
+ * SUT: {@link Const.LegacyURIs#INSTRUCTOR_COURSE_JOIN}.
+ */
+public class InstructorCourseJoinConfirmationPageE2ETest extends BaseE2ETestCase {
+    InstructorAttributes newInstructor;
+
+    @Override
+    protected void prepareTestData() {
+        testData = loadDataBundle("/InstructorCourseJoinConfirmationPageE2ETest.json");
+        removeAndRestoreDataBundle(testData);
+
+        newInstructor = testData.instructors.get("ICJConfirmationE2eT.instr.CS1101");
+        newInstructor.googleId = "ICJConfirmationE2eT.instr2";
+    }
+
+    @BeforeClass
+    public void classSetup() {
+        browser.driver.manage().deleteAllCookies();
```

Comment:
I'm aware that this is from previous UI test, but is this necessary? I don't think cookies are carried over to browsers initialized by Selenium.

Reply:
I think it may be needed for testing with staging server as user info is copied from tester's browser.